### PR TITLE
Bessel example translation to python3 and Jupyter notebook

### PR DESCRIPTION
**Pull request by juanfe**

### CODE REVIEW

File: tutorials/math/Bessel.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+gPad.SetFrameFillColor(19)
+leg = TLegend(0.75, 0.7, 0.89, 0.89)
+
+# Drawing the set of Bessel J functions
+#
+# n is the number of functions in each pad
+n = 5
+JBessel = []
+for nu in range(n):
+    _jbessel = TF1("J_0", "ROOT::Math::cyl_bessel_j([0],x)", 0, 10)
```
Comment (dpiparo) [inline tutorials/math/Bessel.py]: why this name "_jbessel"?

File: tutorials/math/Bessel.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+##
+## \macro_image
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TCanvas, TF1, gSystem, gPad, TLegend, TPaveLabel, kBlack
+
+
+gSystem.Load("libMathMore")
```
Comment (dpiparo) [inline tutorials/math/Bessel.py]: Is this really needed or autoloading can do the job?

File: tutorials/math/Bessel.py (hunk 1 above)
Comment (dpiparo) [reply tutorials/math/Bessel.py]: Can we with the new TFormula avoid params called "[0]"?

File: tutorials/math/Bessel.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+# Set canvas 2
+DistCanvas.cd(2)
+gPad.SetGrid()
+gPad.SetFrameFillColor(19)
+leg2 = TLegend(0.75, 0.7, 0.89, 0.89)
+
+# Drawing Bessel k
+KBessel = []
+for nu in range(n):
+    _kbessel = TF1("J_0", "ROOT::Math::cyl_bessel_k([0],x)", 0, 10)
```
Comment (dpiparo) [inline tutorials/math/Bessel.py]: Same as above for the "[0]".

File: tutorials/math/Bessel.py [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+# Set canvas 3
+DistCanvas.cd(3)
+gPad.SetGrid()
+gPad.SetFrameFillColor(19)
+leg3 = TLegend(0.75, 0.7, 0.89, 0.89)
+
+# Drawing Bessel i
+iBessel = []
+for nu in range(5):
+    _ibessel = TF1("J_0", "ROOT::Math::cyl_bessel_i([0],x)", 0, 10)
```
Comment (dpiparo) [inline tutorials/math/Bessel.py]: same about var name and "[0]".

File: tutorials/math/Bessel.py [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+# Set canvas 4
+DistCanvas.cd(4)
+gPad.SetGrid()
+gPad.SetFrameFillColor(19)
+leg4 = TLegend(0.75, 0.7, 0.89, 0.89)
+
+# Drawing sph_bessel
+jBessel = []
+for nu in range(5):
+    _jbessel = TF1("J_0", "ROOT::Math::sph_bessel([0],x)", 0, 10)
```
Comment (dpiparo) [inline tutorials/math/Bessel.py]: Guess what? same as above ;-)

File: tutorials/math/Bessel.py (hunk 2 above)
Comment (juanfe) [reply tutorials/math/Bessel.py]: Doesn't work with out this line.

File: tutorials/math/Bessel.py (hunk 1 above)
Comment (juanfe) [reply tutorials/math/Bessel.py]: Do you refer to use TFunction instead TF1? I don't understand. TFunction as a I saw receive analitical functions, but in this case the bessel function is predefined in ROOT::Math::cyl_bessel_

**Comment by etejedor** [comment]
@phsft-bot build

**Pull request merged**